File: replace_colors.php
<?php
$directory = 'c:\xampp\htdocs\homeservices-12Mar2026';
$colors_to_replace = ['/#6777ef/i', '/#2046da/i', '/rgb\(8,\s*124,\s*192\)/i', '/#2a9cf5/i'];
$new_color = '#FE7701';

$iterator = new RecursiveIteratorIterator(new RecursiveDirectoryIterator($directory));
foreach ($iterator as $file) {
    if ($file->isDir()) continue;
    $path = $file->getPathname();
    if (strpos($path, 'node_modules') !== false || strpos($path, 'vendor') !== false || strpos($path, '.git') !== false) {
        continue;
    }
    $ext = pathinfo($path, PATHINFO_EXTENSION);
    if (in_array($ext, ['css', 'scss', 'js', 'php'])) {
        $content = file_get_contents($path);
        $changed = false;
        foreach ($colors_to_replace as $color) {
            if (preg_match($color, $content)) {
                $content = preg_replace($color, $new_color, $content);
                $changed = true;
            }
        }
        if ($changed) {
            file_put_contents($path, $content);
            echo "Updated $path\n";
        }
    }
}
echo "Done\n";

File: replace_colors.py
import os
import re

directory = r"c:\xampp\htdocs\homeservices-12Mar2026"
colors_to_replace = [r'#6777ef', r'#6777EF', r'#2046da', r'#2046DA', r'rgb\(8, 124, 192\)', r'#2a9cf5']
new_color = '#FE7701'

for root, dirs, files in os.walk(directory):
    if 'node_modules' in root or 'vendor' in root or '.git' in root:
        continue
    for file in files:
        if file.endswith('.css') or file.endswith('.scss') or file.endswith('.js') or file.endswith('.php'):
            filepath = os.path.join(root, file)
            try:
                with open(filepath, 'r', encoding='utf-8') as f:
                    content = f.read()
                
                changed = False
                for color in colors_to_replace:
                    if re.search(color, content):
                        content = re.sub(color, new_color, content)
                        changed = True
                
                if changed:
                    with open(filepath, 'w', encoding='utf-8') as f:
                        f.write(content)
                    print(f"Updated {filepath}")
            except Exception as e:
                print(f"Could not process {filepath}: {e}")

print("Done")
